### ISSUE TITLE
variable declaration causes handlePastedText function error after js minified

### ISSUE THREAD

**Issue by june50232** [jpuri/react-draft-wysiwyg#621]
check out **var o** below:
before minified:
```
this.handlePastedText = function(t, n) {
    if (e.props.handlePastedText && e.props.handlePastedText(t, n, o, e.onChange))
         return !0;
    var o = e.state.editorState;
    return (0,
    x.handlePastedText)(t, n, o, e.onChange)
 }    
```           

after minified:
```
 this.handlePastedText = function(t, n) {
    return e.props.handlePastedText ? e.props.handlePastedText(t, n, editorState, 
    e.onChange) : !e.props.stripPastedStyles && (0,
    O.handlePastedText)(t, n, e.state.editorState, e.onChange)
}
```

this leads to be unable pasting texts and a console error:
```
react.sns2_bundle.js:formatted:81317 Uncaught ReferenceError: editorState is not defined
    at Object.Q.handlePastedText (react.sns2_bundle.js:formatted:81317)
    at r (react.sns2_bundle.js:formatted:76245)
    at react.sns2_bundle.js:formatted:74574
    at Object.r (react.sns2_bundle.js:formatted:19538)
    at a (react.sns2_bundle.js:formatted:19327)
    at Object.s [as executeDispatchesInOrder] (react.sns2_bundle.js:formatted:19337)
    at p (react.sns2_bundle.js:formatted:15367)
    at m (react.sns2_bundle.js:formatted:15374)
    at Array.forEach (<anonymous>)
    at r (react.sns2_bundle.js:formatted:24598)
```

**Comment by june50232** [comment]
i believe same error related to  https://github.com/jpuri/react-draft-wysiwyg/issues/619

**Comment by Huespal** [comment]
Same here. 
```
Uncaught ReferenceError: editorState is not defined
    at Object.J.handlePastedText (bundle.js?v=release-1.37.0:formatted:76224)
```

**Comment by Huespal** [comment]
I solved. I created a Pull request. 
If want to solve it by yourself, the bug is in Editor/index.js 
Just move up editorState variable declaration, as @june50232 suggested.

in `Editor/index.js` code has to be like this:

```
handlePastedText = (text, html) => {
    const { editorState } = this.state;

    if (this.props.handlePastedText) {
      return this.props.handlePastedText(
        text,
        html,
        editorState,
        this.onChange
      );
    }
    if (!this.props.stripPastedStyles) {
      return handlePastedText(text, html, editorState, this.onChange);
    }
    return false;
  };
```

**Comment by jpuri** [comment]
Change has been released `react-draft-wysiwyg@1.12.11`. Thanks again 👍
